includes/blocks/course-list/class-sensei-course-list-categories-filter.php

```diff
@@ -41,7 +42,7 @@ public function get_content( $query_id ) : string {
 		);
 
 		return '<select class="course_list_filter_block" data-param-key="' . $this->param_key . '" data-query-id="' . $query_id . '" >
```

Comment:
Sorry, I just noticed this after the fact. The CSS class naming is not what we typically use elsewhere for Sensei blocks.

If we look at some of our other blocks (e.g. course outline), then we see that we typically use the CSS prefix of the block, followed by some modifier. For example, here we're naming the class `course_list_filter_block`. Following the course outline block's example, it should be something like `wp-block-sensei-lms-course-list-filter__category`.

Another rule we try to follow is to match the editor CSS class names with the frontend CSS class names as best we can so that both views are easier to maintain.

Having said all of that, after searching the codebase I can't find where this class is being used. Do you think it can be removed?

Relatedly, in the HTML the data parameters look a little weird (they all end with a dash):

![Screen Shot 2022-09-08 at 5 31 30 PM](https://user-images.githubusercontent.com/1190420/189229783-c0711df3-7ee5-4379-bbbe-3edafc969891.jpg)

Reply:
Thanks a lot for the explanation @donnapep :) This class wasn't being used anymore. Removed it and also changed the params [here](https://github.com/Automattic/sensei/pull/5578/commits/6dd7e20a4ec1957e3be81aa72eaa758747e3b25f)